src/Engine.php

```diff
@@ -12,47 +12,23 @@
  */
 class Engine
```

Comment:
:+1: Each core Class should have an interface. Modules can later depend on that Interface which allows for different Engines. I have no use case for that ATM, but Userland is so crazy. Who knows :8ball: